### PR TITLE
Update instructions and tests of change font-size challenge

### LINKED ISSUE #6819: Mismatched error message for Font Size of an Element
Challenge [Change the Font Size of an Element](http://www.freecodecamp.com/challenges/change-the-font-size-of-an-element) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.111 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```html
<style>
  p {
    font-size: 16px;
  }
  .red-text {
    color: red;
  }
</style>

<h2 class="red-text">CatPhotoApp</h2>

<p class="red-text">Kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff.</p>

<p class="red-text">Purr jump eat the grass rip the couch scratched sunbathe, shed everywhere rip the couch sleep in the sink fluffy fur catnip scratched.</p>
```
Having `class="red-text"` on the second p tag gives the error message:
**Give elements with the p tag a font-size of 16px. Browser and Text zoom should be at 100%.
screen shot 2016-02-07 at 7 12 11 pm
**
I kept trying different zooms and browsers and code order and got the same message. If I change the second p tag to `<p>Purr jump eat the grass rip the couch scratched sunbathe, shed everywhere rip the couch sleep in the sink fluffy fur catnip scratched.</p>` the code passes.

It is confusing that the error message is about font size when the real problem is the color of the text.

### PR DESCRIPTION
**Pull request by ankitaggarwal011**
Update instructions and test cases of Challenge [Change the Font Size of an Element](http://www.freecodecamp.com/challenges/change-the-font-size-of-an-element).

closes #6819.

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/html5-and-css.json [hunk 1]
```diff
@@ -477,7 +477,7 @@
         "<blockquote>h1 {<br>&nbsp;&nbsp;font-size: 30px;<br>}</blockquote>",
         "Create a second <code>p</code> element with the following kitty ipsum text: <code>Purr jump eat the grass rip the couch scratched sunbathe, shed everywhere rip the couch sleep in the sink fluffy fur catnip scratched.</code>",
         "Inside the same <code>&#60;style&#62;</code> tag that contains your <code>red-text</code> class, create an entry for <code>p</code> elements and set the <code>font-size</code> to 16 pixels (<code>16px</code>).",
-        "<strong>Note</strong><br>Due to browser implementation differences, you may need to be at 100% zoom to pass the tests on this challenge.<br>Also, please do not add any extra classes to your new element."
+        "<strong>Note</strong><br>Due to browser implementation differences, you may need to be at 100% zoom to pass the tests on this challenge.<br><br>Also, please do not add class attribute to your new <code>p</code> element."
```
Comment (SaintPeter) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: Grammar:
> Please do not add **a** class attribute to your new . . .

Comment (SaintPeter) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: Also, if you want a double line break, just remove the `<br><br>` entirely and break this into two lines.
```js
"<strong>Note</strong><br>Due to browser implementation differences, you may need to be at 100% zoom to pass the tests on this challenge.",
"Also, please do not add class attribute to your new <code>p</code> element."
```

**Comment by SaintPeter** [comment]
Aside from the above notes, I tested this and it works well.  You can `git commit --amend` to update this PR - just do a `git push -f` to update it.

**Comment by ankitaggarwal011** [comment]
@SaintPeter, Done! Thanks for the help.

**Comment by QuincyLarson** [comment]
@ankitaggarwal011 thanks for your contribution. This works great!

**Pull request merged**